29480 Template management

tyndmar opened the pull request:
Files changed:
addon\touch\r-timesheet.p
addon\touch\r-tsxl.p
arrep\asiexlstmt.p
ar\rep\invpryst-xl.p
ar\rep\invsthpk-xl.p
cec\quote\quobell-xl.p
cec\quote\quoccc-xl.p
cec\quote\quocsc-xl.p
cec\quote\quofib-xl.p
cec\quote\quoknight-xl.p
cec\quote\quomsp-xl.p
cec\quote\quoprm-mci.p
cec\quote\quoprm-xl.p
cec\quote\quotri-xl.p
cerep\jobindxl-rs.p
cerep\jobpryst.p
cerep\jobtickindxl.p
custom\MoveExcelT.w
est\specprem.p
oerep\fibreci_MEX-XL.p
oerep\trkschxl.p
oe\rep\ackalbert.p
oe\rep\bolpryst-xl.p
oe\rep\bolsouth-xl.p
oe\rep\cocacpi.p
oe\rep\cocccc.p
oe\rep\cocuni.p
oe\rep\cocwpp.p
oe\rep\invpryst-xl.p
oe\rep\invsthpk-xl.p
pcrep\r-prodlyN.w
po\po-sthpk-xl.p
Ensure that no hard code or search finds /Template in environment (must go to /custfiles or /resources)

Commit:
29480 Template management
Files changed:
addon\touch\r-timesheet.p
addon\touch\r-tsxl.p
arrep\asiexlstmt.p
ar\rep\invpryst-xl.p
ar\rep\invsthpk-xl.p
cec\quote\quobell-xl.p
cec\quote\quoccc-xl.p
cec\quote\quocsc-xl.p
cec\quote\quofib-xl.p
cec\quote\quoknight-xl.p
cec\quote\quomsp-xl.p
cec\quote\quoprm-mci.p
cec\quote\quoprm-xl.p
cec\quote\quotri-xl.p
cerep\jobindxl-rs.p
cerep\jobpryst.p
cerep\jobtickindxl.p
custom\MoveExcelT.w
est\specprem.p
oerep\fibreci_MEX-XL.p
oerep\trkschxl.p
oe\rep\ackalbert.p
oe\rep\bolpryst-xl.p
oe\rep\bolsouth-xl.p
oe\rep\cocacpi.p
oe\rep\cocccc.p
oe\rep\cocuni.p
oe\rep\cocwpp.p
oe\rep\invpryst-xl.p
oe\rep\invsthpk-xl.p
pcrep\r-prodlyN.w
po\po-sthpk-xl.p
Ensure that no hard code or search finds /Template in environment (must go to /custfiles or /resources)

brad-advantzware reviewed (changes requested):
2 instances of hard-coded strings per file.  Need to use variable reference to this string in messages.

Commit:
29480 Template management
Files changed: see list
removed all unique references to locate a template in favor of new program sys/ref/getFileFullPathname.p
getFileFullPathname.p
accepts input parameter of a relative or fully-qualifed pathname, e.g. "template\tmplate.xlt" or "n:\admin\advantzware.ini"
verifies that the file can be found (in the propath if relative, or in the filesystem if qualified)
returns the fully qualified file name, e.g "n:\environments\prod\custfiles\templates\template.xlt"
if NOT found, displays error message and returns null value ("?") as char string

Commit:
Update quomsp-xl.p

Commit:
Update jobindxl-rs.p

brad-advantzware reviewed (approved):
DIscussed other refactoring possible with Mark, but this is good for now
